tests/tt_eager/python_api_testing/unit_testing/backward_ops/test_backward_mul.py

```diff
@@ -61,23 +61,23 @@ def test_bw_mul_opt_output(input_shapes, device, are_required_outputs, pass_queu
 
     cq_id = 0
     if pass_queue_id:
-        tt_output_tensor_on_device = tt_lib.tensor.mul_bw(
+        tt_output_tensor_on_device = ttnn.mul_bw(
             grad_tensor,
             input_tensor_a,
             input_tensor_b,
             are_required_outputs=are_required_outputs,
-            input_a_grad=input_a_grad,
-            input_b_grad=input_b_grad,
+            optional_input_a_grad=input_a_grad,
+            optional_input_b_grad=input_b_grad,
```

Comment:
imho, these arguments should not have "optional" in their name. The fact that they are kw_only is enough to reflect that.
Its ok to keep it like that in this PR if it is not added in this PR, but lets make sure we address it in future changes.